README.md

```diff
@@ -5,6 +5,9 @@
 ![Tests](https://github.com/mackelab/sbi/workflows/Tests/badge.svg?branch=main)
 
 ## sbi: simulation-based inference
+[Getting Started](https://www.mackelab.org/sbi/tutorial/00_getting_started/) |
+[Documentation](https://www.mackelab.org/sbi/) |
+[Contributing](https://github.com/mackelab/sbi/blob/main/CONTRIBUTING.md)
```

Comment:
I would prefer a badge for `Documentation` (not so sure how easily this can be done, we do not use readthedocs and I am not sure if mkdocs supports a badge). Not sure if we need the other two...were they explicitly requested by the reviewers?

Reply:
No, I just followed https://github.com/pyro-ppl/pyro

Reply:
Ah, ok, cool. From the raw markdown I had missed that they would get displayed next to each other. Happy with all three links then :)

Reply:
And I couldn't find a badge for mkdocs.